esx_service/cli/vmdkops_admin.py

```diff
@@ -920,6 +920,8 @@ def generate_tenant_ls_rows(tenant_list):
             default_datastore = ""
         else:
             default_datastore = vmdk_utils.get_datastore_name(tenant.default_datastore_url)
+            if default_datastore is None:
```

Comment:
nit: Would be nice to comment... something like "use empty string to simplify line formatting down the road"